src/math/operations/cutPasteOps/augment.ts

```diff
@@ -6,7 +6,17 @@ import { Polyhedron, Face, Edge, FaceLike } from "math/polyhedra"
 import { repeat, find } from "utils"
 import { makeOperation } from "../Operation"
 import { deduplicateVertices, alignPolyhedron, Pose } from "../operationUtils"
-import { inc, CutPasteSpecs, CutPasteOpArgs, combineOps } from "./cutPasteUtils"
+import {
+  CutPasteSpecs,
+  CutPasteOpArgs,
+  combineOps,
+  augDimCapstoneGraph,
+  augDimAugmentedSolidGraph,
+  augDimDiminishedSolidGraph,
+  augDimGyrateSolidGraph,
+  augDimElementaryGraph,
+} from "./cutPasteUtils"
```

Comment:
I can probably make a binary augment/diminish pair and then just have augment and diminish on the same page.